.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Release_Charts'
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "${{ secrets.GIT_USERNAME }}"
+          git config user.email "${{ secrets.GIT_USEREMAIL }}"
```

Comment:
Are these provided by github directly in the action context or do these need to be provided through repo configuration? If they are not provided are there alternative provided through github actions that can be used?

Reply:
these variables have to be provided through repo configuration.